### PR TITLE
Improve CLI docs

### PR DESCRIPTION
**Pull request by riedgar-ms**
Have a ready-to-go YAML in the CLI examples. Hook this into the testing infrastructure, and also update the instructions on using it. Not to mention the name of the instructions doc itself.

### CODE REVIEW

File: examples/CLI/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Quickstart - AzureML CLI Model Analysis
```
Comment (minthigpen) [inline examples/CLI/ReadMe.md:1]: We shouldn't use "Model analysis" -- instead "Azure ML CLI Responsible AI Insights"

File: examples/CLI/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Prequisites
+To run the CLI and create a model analysis, make sure that you have completed the setup instructions per the below link.
+
+-  [Setup instructions](https://github.com/Azure/RAI-vNext-Preview#set-up)
+
+After following the setup instructions above, your AzureML workspace should have all the required RAI and training components in your workspace to train a simple model and run a model analysis on it. Next, we want to create a pipeline that will execute these components in order to create a model and model analysis.
```
Comment (minthigpen) [inline examples/CLI/ReadMe.md:8]: Replace all "model analysis" with "Responsible AI insights"

**Review by minthigpen — APPROVED** [review]

File: examples/CLI/ReadMe.md (hunk 1 above)
Comment (riedgar-ms) [reply examples/CLI/ReadMe.md:1]: Done

File: examples/CLI/ReadMe.md (hunk 2 above)
Comment (riedgar-ms) [reply examples/CLI/ReadMe.md:8]: Done.

**Pull request merged**